R/plotFeatureLoadings.R

```diff
@@ -0,0 +1,404 @@
+#' Plot tree with feature loadings after performing a reducedDim (e.g. PCA, LDA, NMF)
+#'
+#' Based on the \code{\link[diffTop:plotASVcircular]{plotASVcircular}} method using phyloseq 
+#' and has been converted to use TreeSummarizedExperiment objects.
+#'
+#' @param object a
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'   object.
+#' 
+#' @param method One dimensionality reduction method of \code{c("PCA", "LDA", "NMF")}
+#'   indicating which method will be used.
+#'   (default: \code{method = "PCA"})
+#'  
+#' @param rank a single character defining a taxonomic rank. Must be a value of
```

Comment:
Btw is it possible to show tree and PC components with higher resolution than the colored rank info?

Reply:
I think this should be possible